cli/options_test.go

```diff
@@ -308,3 +309,16 @@ func TestEnvMap(t *testing.T) {
 	m = utils.GetAsEqualsMap(l)
 	assert.Equal(t, m["foo"], "bar")
 }
+
+func TestGetEnvFromFile(t *testing.T) {
+	wd := t.TempDir()
+	f := filepath.Join(wd, ".env")
+	err := os.Mkdir(f, 0700)
```

Comment:
```suggestion
	err := os.Mkdir(f, 0o700)
```
to make the linter happy